src/handlers/shared/start.ts

```diff
@@ -37,21 +38,18 @@ export async function start(context: Context, issue: Context["payload"]["issue"]
   // check max assigned issues
 
   const openedPullRequests = await getAvailableOpenedPullRequests(context, sender.login);
-  logger.info(`Opened Pull Requests with approved reviews or with no reviews but over 24 hours have passed: ${JSON.stringify(openedPullRequests)}`);
+  logger.info(`Opened Pull Requests with approved reviews or with no reviews but over 24 hours have passed`, {
+    openedPullRequests: openedPullRequests.map((i) => i.html_url),
+  });
 
   const assignedIssues = await getAssignedIssues(context, sender.login);
-  logger.info("Max issue allowed is", { maxConcurrentTasks });
+  logger.info("Max issues allowed is", { limit: maxTask.limit, assigned: assignedIssues.length });
 
   // check for max and enforce max
 
-  if (assignedIssues.length - openedPullRequests.length >= maxConcurrentTasks) {
-    const log = logger.error("Too many assigned issues, you have reached your max limit", {
-      assignedIssues: assignedIssues.length,
-      openedPullRequests: openedPullRequests.length,
-      maxConcurrentTasks,
-    });
-    await addCommentToIssue(context, log?.logMessage.diff as string);
-    throw new Error(`Too many assigned issues, you have reached your max limit of ${maxConcurrentTasks} issues.`);
+  if (Math.abs(assignedIssues.length - openedPullRequests.length) >= maxTask.limit) {
+    await addCommentToIssue(context, "```diff\n! Too many assigned issues, you have reached your max limit.\n```");
```

Comment:
I noticed that the logger is being used mostly where the error message has variables in it. And in a couple of cases across the code the logger is not used where there a no variables to pass in the error message. So I think the trend is just being followed here.

Reply:
My mistake @gentlementlegen I think this may have been me, still forming the habit

@jordan-ae variables don't play into it, the logger takes the log and creates 4 versions of it, one of them is `diff` which is how the bot comments most of the time.

```ts
await addCommentToIssue(context, log?.logMessage.diff as string);
```

is the correct pattern